README.rst

```diff
@@ -95,8 +102,8 @@ Settings
 DRAMATIQ_BROKER
   A dict of options to pass when instantiating the broker.
 
-DRAMATIC_BROKER['OPTIONS']
-  Arguments to pass to the Broker.
```

Comment:
Did you deliberately remove this?

Reply:
Yes, this is not necessary anymore because we use the alias to the Django database connection instead of a custom URL

Reply:
And you believe it's not possible to want to pass any other options to the Broker?

Reply:
I just assumed the intention based on the code and documentation.
Let me go through the available broker options.

```python
pool=None,
url="",
results=True,
schema=None,
table=None,
middleware=None
```

1. `pool` should be a `ThreadedConnectionPool` instance that is impossible without additional code just pass as a param
2. `url` we construct from `DATABASE_ALIAS`
3.  `schema` and `table` are already hardcoded in the code in the `BackgroundJob.Meta.db_table` as default broker values (`dramatiq` and `queue` respectively) and previously they were hardcoded as `BackgroundJob.Meta.db_table` and literal in the signal handler. So passing the different values would cause the problem without changing the code of the migration and SQL script
4. `middleware` parsed from the `MIDDLEWARE`
5. `results` is used to configure `Results` results middleware with `PostgresBackend`. This can be achieved via `MIDDLEWARE` or left `result=True` as it is.

As the result, the only meaningful and available option for the broker is the `results` variable which can be:
* left as is
* placed in a separate variable
* added as separate middleware

Reply:
@funkybob Did I address your concerns?